src/test_case.js

```diff
@@ -91,8 +97,13 @@ TestCase.prototype.build = function (parentElement) {
   if (this._standardOutput) {
     testCaseElement.ele('system-out').cdata(this._standardOutput);
   }
+  var systemError = undefined;
   if (this._standardError) {
-    testCaseElement.ele('system-err').cdata(this._standardError);
+    systemError = testCaseElement.ele('system-err').cdata(this._standardError);
+
+    if(this._errorAttachment) {
+      systemError.txt(`[[ATTACHMENT|${this._errorAttachment}]]`);
```

Comment:
Ok, I've fixed the original error on `master`, by pinning a development dependency, but the backticks now fails the build on old versions (`0.10`, `0.12`). While I like the syntax, I think I would like to keep this compatible with old versions as well. Does that make sense?